packages/kbn-test/src/functional_test_runner/lib/mocha/reporter/reporter.js

```diff
@@ -23,12 +23,12 @@ import Mocha from 'mocha';
 import { ToolingLogTextWriter } from '@kbn/dev-utils';
 import moment from 'moment';
 
-import { setupJUnitReportGeneration } from '../../../../dev';
+import { setupJUnitReportGeneration } from '../../../../../../../src/dev';
```

Comment:
This type of import, that leaves the `@kbn/test` package (goes into `src/dev` in this instance), is something we'll need to either duplicate into `@kbn/test` or just move into here. For the JUnit stuff we should probably just move all the reporters into `@kbn/test`, but we should do that in a separate PR.

Reply:
Ok, so it sounds like we want to move `src/dev/mocha`'s contents into `kbn-test`.  Do we want that refactor in this PR, or a follow on?

Reply:
Separate PR